src/backends.rs

```diff
@@ -38,6 +38,9 @@ cfg_if! {
                 );
             }
         }
+    } else if #[cfg(all(target_os = "linux", target_env = ""))] {
```

Comment:
Technically, we only have the `x86_64-unknown-linux-none` target, but to future-proof ourselves I think it's fine to use a more general `cfg`.